title_belt_nhl/tests/test_schedule.py

```diff
@@ -0,0 +1,15 @@
+import json
+from title_belt_nhl.main import Schedule
+
+def test_current_title_belt_holder():
+    mock_data_path = './title_belt_nhl/tests/test_files/mock_league_schedule.json'
+
+    # Open the file and load the JSON data
+    with open(mock_data_path, 'r') as file:
+        leagueSchedule = json.load(file)
```

Comment:
This isn't a bad way to do it, but you could also make the JSON array a pytest fixture in conftest.py inside `tests/`:

```py
@pytest.fixture
def league_schedule():
    return [
      {
        "id": 2023010045,
        "season": 20232024,
        # ... all the other stuff
      }
    ]
```
And then in the tests you can just say
```py
def test_current_title_belt_holder(league_schedule):
    # use league_schedule the same way you're using `leagueSchedule` below)
```
because the fixtures defined in conftest.py are available to any tests in the directory (you just pass them by name): https://docs.pytest.org/en/stable/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files